examples/gslib/exp.cpp

```diff
@@ -0,0 +1,211 @@
+﻿//          Parallel example of utilizing GSLib's FindPoints methods
+//
+// Compile with: make exp
+//
+// Sample runs:
+//    mpirun -np 2 exp -m ../../data/rtaylor2D-q3.mesh -o 3
+//    mpirun -np 2 exp -m ../../data/fichera.mesh -o 3
+
+#include "mfem.hpp"
+
+using namespace mfem;
+using namespace std;
+
+double field_func(const Vector &x)
+{
+   const int dim = x.Size();
+   double res = 0.0;
+   for (int d = 0; d < dim; d++) { res += x(d) * x(d); }
+   return res;
+}
+
+int main (int argc, char *argv[])
+{
+   // Initialize MPI.
+   int num_procs, myid;
+   MPI_Init(&argc, &argv);
+   MPI_Comm_size(MPI_COMM_WORLD, &num_procs);
+   MPI_Comm_rank(MPI_COMM_WORLD, &myid);
+
+   // Set the method's default parameters.
+   const char *mesh_file = "RT2D.mesh";
```

Comment:
This mesh file doesn't exist at the moment...

Reply:
I have changed the mesh name.